GitCommands/Git/GitRevision.cs

```diff
@@ -118,9 +118,20 @@ public static bool IsArtificial(string guid)
                     guid == IndexGuid;
         }
 
-        public bool HasParent()
+        public bool HasParent
         {
-            return ParentGuids != null && ParentGuids.Length > 0;
+            get
+            {
+                return ParentGuids != null && ParentGuids.Length > 0;
+            }
+        }
+
+        public string GetParentGuid
```

Comment:
the name is bad - the property is a "get", so calling it "GetXxx" is a duplication.
I'd call this something like `Parent` or even `FirstParentGuid`